lib/internal/child_process/serialization.js

```diff
@@ -15,6 +18,10 @@ const kMessageBuffer = Symbol('kMessageBuffer');
 const kJSONBuffer = Symbol('kJSONBuffer');
 const kStringDecoder = Symbol('kStringDecoder');
 
+const { subarray } = Uint8ArrayPrototype;
+const Uint8ArrayPrototypeSubarray =
+  (thisBuffer, ...args) => ReflectApply(subarray, thisBuffer, args);
```

Comment:
There is probably a better way to do this, but I wasn't able to figure it out. @targos maybe you know why `Uint8ArrayPrototypeSubarray` is not available on `primordials`?

Reply:
Because it’s not an own property of `Uint8Array.prototype`.

Reply:
I see, maybe I should name it `TypedArrayPrototypeSubarray` in `primordials.js`.